aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIOIntegration.java

```diff
@@ -80,26 +82,32 @@ public class TestS3FileIOIntegration {
   private static byte[] contentBytes;
   private static String content;
   private static String kmsKeyArn;
+  private static String kmsKeyAlias;
   private static int deletionBatchSize;
   private String objectKey;
   private String objectUri;
 
   @BeforeClass
   public static void beforeClass() {
-    clientFactory = AwsClientFactories.defaultFactory();
+    clientFactory = AwsClientFactories.from(
+        ImmutableMap.of(AwsProperties.HTTP_CLIENT_TYPE, AwsProperties.HTTP_CLIENT_TYPE_APACHE));
     s3 = clientFactory.s3();
     kms = clientFactory.kms();
     s3Control = AwsIntegTestUtil.createS3ControlClient(AwsIntegTestUtil.testRegion());
     crossRegionS3Control = AwsIntegTestUtil.createS3ControlClient(AwsIntegTestUtil.testCrossRegion());
     bucketName = AwsIntegTestUtil.testBucketName();
     crossRegionBucketName = AwsIntegTestUtil.testCrossRegionBucketName();
-    accessPointName = UUID.randomUUID().toString();
-    crossRegionAccessPointName = UUID.randomUUID().toString();
+    accessPointName = "icebergawsci-" + UUID.randomUUID();
+    crossRegionAccessPointName = "icebergawsci-" + UUID.randomUUID();
     prefix = UUID.randomUUID().toString();
     contentBytes = new byte[1024 * 1024 * 10];
     deletionBatchSize = 3;
     content = new String(contentBytes, StandardCharsets.UTF_8);
     kmsKeyArn = kms.createKey().keyMetadata().arn();
+    kms.createAlias(CreateAliasRequest.builder()
+        .targetKeyId(kmsKeyArn)
+        .aliasName("alias/iceberg-aws-ci-" + UUID.randomUUID())
+        .build());
```

Comment:
is it required ? I see kmsKeyAlias declared above as well but it neither assigned this value nor used